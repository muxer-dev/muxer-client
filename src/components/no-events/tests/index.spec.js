import React from "react";
import NoEvents from "../";

describe("no-events", function() {
  it("renders", function() {
    expect(<NoEvents location={"belfast"}/>).toMatchSnapshot("location");
  });
});
