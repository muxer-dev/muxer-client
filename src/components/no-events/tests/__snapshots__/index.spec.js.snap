// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`location 1`] = `
<NoEvents
  className=""
  location="belfast"
/>
`;
